engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -324,9 +324,11 @@ object ProgramExecutionSupport {
     ctx: RuntimeContext
   ): PartialFunction[Throwable, Api.ExecutionResult.Diagnostic] = {
     case ex: AbstractTruffleException
+        // ExitException is special, and handled differently than other TruffleExceptions.
+        if !ex.isInstanceOf[ExitException] &&
```

Comment:
I assume we want the same handling for "exit exceptions" that are coming from other languages - Python for example. In such case I suggest to avoid `isInstanceOf`, but rather use `InteropLibrary.getUncached()` to query the type of `AbstractTruffleException`.

Reply:
Remove `isInstanceOf` check and replaced with `InteropLibrary.isExitException` message in 3385e4ccb3952438baafd4ab1f2fa89e6439648e